### ISSUE TITLE
Problem with pullURL for ontologies in @Web

### ISSUE THREAD

**Issue by antool** [agroportal/agroportal_web_ui#128]
Lorqu'une ontologie est téléchargée sur le portail @web de l'INRA ([http://pfl.grignon.inra.fr/atWeb/](url)), le fichier est généré à la volée et la métadonnée `<owl:versionInfo>` est automatiquement complétée avec la date du jour en cours.
Ainsi, lorsqu'une pullURL pointe sur ce site (par ex. pour Biorefinery ou Transmat), un changement est détecté chaque nuit dans le fichier de l'ontologie par AgroPortal (il y a à chaque fois une nouvelle valeur dans le champ `<owl:versionInfo>`) et provoque une nouvelle soumission. Ceci explique le nombre important de soumissions pour ces ontologies alors qu'il n'y a en fait que de très rares véritables modifications dans le fichier.

**Comment by antool** [comment]
Biorefinery et Transmat soumises à nouveau avec un historique "raisonnable" des versions et sans pullURL.

**Comment by selios** [comment]
Oui, la plateforme atWeb continuera d'exister en cohabitation avec le Dataverse. Les développement sont déjà en cours sur atWeb pour envoyer les métadonnées des tableaux annotés sur le dataverse avec un lien redirigeant vers la ressource correspondante sur atWeb.
https://data.inra.fr/dataverse/atweb

**Comment by antool** [comment]
@selios Est-il envisagé de changer le système de téléchargement des ontologies sur atweb pour régler le problème de la pullURL décrit ci-dessus  (i.e.  ne pas générer une `<owl:versionInfo>` différente chaque jour) ?
